### PR TITLE
add consistent hash bangs to scripts

### PR DESCRIPTION
**Pull request by maleck13**

### CODE REVIEW

**Review by luksa — APPROVED** [review]

**Comment by luksa** [comment]
Thanks @maleck13

File: api/v1alpha1/istio_types.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ type IstioSpec struct {
 	// Must be one of: ambient, default, demo, empty, external, minimal, openshift, preview, remote.
 	// +++PROFILES-DROPDOWN-HIDDEN-UNTIL-WE-FULLY-IMPLEMENT-THEM+++operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Profile",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:fieldGroup:General", "urn:alm:descriptor:com.tectonic.ui:select:ambient", "urn:alm:descriptor:com.tectonic.ui:select:default", "urn:alm:descriptor:com.tectonic.ui:select:demo", "urn:alm:descriptor:com.tectonic.ui:select:empty", "urn:alm:descriptor:com.tectonic.ui:select:external", "urn:alm:descriptor:com.tectonic.ui:select:minimal", "urn:alm:descriptor:com.tectonic.ui:select:preview", "urn:alm:descriptor:com.tectonic.ui:select:remote"}
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
-	// +kubebuilder:validation:Enum=ambient;default;demo;empty;external;minimal;openshift;preview;remote
+	// +kubebuilder:validation:Enum=ambient;default;demo;empty;external;minimal;openshift;preview;remote;
```
Comment (jwendell) [inline api/v1alpha1/istio_types.go]: Is this related? IIRC this line is automatically generated, we should not edit it directly, right @luksa ?

File: Makefile.core.mk [hunk 2]
```diff
@@ -307,6 +307,7 @@ update-istio: ## Updates the Istio commit hash in the 'latest' entry in versions
 ## Location to install dependencies to
 LOCALBIN ?= $(shell pwd)/bin
 $(LOCALBIN):
+	echo "mkdir -p $(LOCALBIN)"
```
Comment (jwendell) [inline Makefile.core.mk]: This is redundant, I think. The command below is echoed anyway because there's no `@` at the beginning to mute it.

File: api/v1alpha1/istio_types.go (hunk 1 above)
Comment (maleck13) [reply api/v1alpha1/istio_types.go]: this was added automatically when I ran one of the make commands. I can remove it but I didn't add directly

File: Makefile.core.mk (hunk 2 above)
Comment (maleck13) [reply Makefile.core.mk]: yep np missed that one
